### PR TITLE
Add recipes for new packages bcolz-zipline and zipline-reloaded

### PR DESCRIPTION
**Pull request by stefan-jansen**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by stefan-jansen** [comment]
Happy to be a maintainer for both packages.

**Comment by stefan-jansen** [comment]
Impressive what you've put together here, this seems to work really well (fingers crossed..).

**Comment by stefan-jansen** [comment]
@conda-forge/help-python This recipe has successfully passed all the checks and is ready to be reviewed. Thanks in advance!

File: recipes/bcolz-zipline/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    - python
+    - setuptools-scm
+    - toml
+    - {{ pin_compatible('numpy') }}
+  run:
+    - python
+
+test:
+  imports:
+    - bcolz
```
Comment (dopplershift) [inline recipes/bcolz-zipline/meta.yaml:33]: Does that mean this package completely conflicts with the normal `bcolz` package?

Comment (stefan-jansen) [reply recipes/bcolz-zipline/meta.yaml:33]: It's an updated version since the original is unmaintained and it's needed for the updated version of zipline.

**Pull request closed without merging**